guides/source/action_cable_overview.md

```diff
@@ -212,11 +213,6 @@ function getWebSocketURL {
 A consumer becomes a subscriber by creating a subscription to a given channel:
 
 ```js
-// app/javascript/channels/chat_channel.js
-import consumer from "./consumer"
-
-consumer.subscriptions.create({ channel: "ChatChannel", room: "Best Room" })
-
 // app/javascript/channels/appearance_channel.js
```

Comment:
Have you inadvertently removed the code example that was here in order to explain it in more detail further on?

Reply:
Oh, I hadn't spotted that you've left `consumer.subscriptions.create({ channel: "AppearanceChannel" })` in the code example and then added the other cases further below. So it looks good.